scripts/uboot_helper

```diff
@@ -316,6 +316,10 @@ devices = \
         'dtb': 'rk3399-nanopi-m4.dtb',
         'config': 'nanopi-m4-rk3399_defconfig'
       },
+      'nanopi-m4-2gb': {
+        'dtb': 'rk3399-nanopi-m4-2gb.dtb',
```

Comment:
Linux only contain `rk3399-nanopi-m4b.dts` and `rk3399-nanopi-m4.dts`, U-Boot does however have a `rk3399-nanopi-m4-2gb.dts`.
This `dtb` option should reference the linux dtb and not the U-Boot dtb.

Reply:
Thank you for pointing this out, I didn't check the Linux mainline source tree before.